src/components/accessories/patientOPD/patientOPDTable/PatientOPDTable.tsx

```diff
@@ -57,8 +81,9 @@ const PatientOPDTable: FunctionComponent<IOwnProps> = ({
     return results;
   };
 
-  const onDelete = () => {
-    console.log("delete");
+  const onDelete = (row: OpdDTO) => {
+    setDeletedObjCode(row.code + "");
```

Comment:
`row.code ?? ""`

Reply:
The objCode to delete is a string. You can look at the initialization. It is just used in translation message.

Reply:
The code `row.code ?? ""` is the same as `row.code + ""` but more javascript compliant.
If the code is undefined then return "".

Reply:
Right. Thanks

Reply:
I applied your suggestions